### PR TITLE
[DOCFIX]在调试向导中增加Alluxio远程调试章节

### PR DESCRIPTION
**Pull request by maobaolong**
在google论坛Alluxio Developers中，我提出了名为“Just Debug it ! Debug it ! Remote Debug with alluxio (eclipse or Idea and others)”的主题，给出了远程调试alluxio的方法，开发者Bin Fan给予了回复，建议我把方法写在调试向导中，这样很多初学者会受益于此。

### CODE REVIEW

**Comment by maobaolong** [comment]
alluxio-bot, check this please

**Comment by RongGu** [comment]
Hey @maobaolong , thanks for this PR. Would you please also add this part into the English documentation? BTW, please express the title and description of this PR in English also. Thanks.

**Comment by maobaolong** [comment]
thank you for your reply，the english version is previous PR, its number is 4537.
the link is https://github.com/Alluxio/alluxio/pull/4537

**Comment by maobaolong** [comment]
sorry ,forget to @RongGu
thank you for your reply，the english version is previous PR, its number is 4537.
the link is https://github.com/Alluxio/alluxio/pull/4537

File: docs/cn/Debugging-Guide.md [hunk 1]
```diff
@@ -19,6 +19,19 @@ Alluxio运行过程中可产生master、worker和client日志，这些日志存
 
 master和worker日志对于理解Alluxio master节点和worker节点的运行过程是非常有帮助的，当Alluxio运行出现问题时，可以查阅日志发现问题产生原因。如果不清楚错误日志信息，可在[邮件列表](https://groups.google.com/forum/#!forum/alluxio-users)查找，错误日志信息有可能之前已经讨论过。
 
+## Alluxio远程调试
+
+Alluxio一般不在开发机上运行,这使得Alluxio的调试变得困难,我们通常会用"增加日志-编译-部署运行-查看日志"的方法来定位问题,而这种定位问题的效率比较低而且需要修改代码从新部署,这在有些时候是不允许的。
+
+使用java远程调试技术可以简单、不修改源码的方式，进行源码级调试。你需要增加jvm 远程调试参数，启动调试服务。增加远程调试参数的方法有很多，比较方便的一种方法是，在需要调试的节点上，修改alluxio-env.sh，增加如下配置属性。
```
Comment (apc999) [inline docs/cn/Debugging-Guide.md]: Same here as my comment in https://github.com/Alluxio/alluxio/pull/4537, 
<pre>`alluxio-env.sh`</pre>

**Comment by apc999** [comment]
One minor comment left. otherwise, LGTM. thanks for this improvement!

**Comment by maobaolong** [comment]
@apc999 
Thanks for apc999's help, your help is careful and useful. Now I accept all your comment.

**Comment by apc999** [comment]
LGTM. thanks for the contribution. Looking forward to seeing you more in our community.

**Pull request merged**